src/atoms/tile/AppTile.ts

```diff
@@ -0,0 +1,41 @@
+import { TTile } from '../../types';
+
+export function AppTile({
+    width,
+    height,
```

Comment:
I recommend on making one of them optional and to make tile with even sides if only one provided

Reply:
Or maybe if one optional other can be calculated based on container/content/etc. Often you really only need one. Maybe also consider custom styling option, on different usecases you may need some different little touches

Reply:
Fixed by added default 'w-full' value for width